homeworks/igor.orel_igor0806/GamesProject/form.css

```diff
@@ -0,0 +1,134 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+body {
+  font-family: "Roboto", sans-serif;
+  color: #221F1F;
+  margin: 0;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 15px 40px;
+  background-color: #EF4934;
+}
+
+.link_list {
+  list-style: none;
+  color: #FFF;
+  display: flex;
+  gap: 50px;
+}
+
+.link:hover {
+  text-decoration: underline;
+}
+
+.nav-link {
+  text-decoration: none;
+  color: #FFF;
+  font-size: 16px;
+  font-weight: 500;
+}
+
+.login_title {
+  text-align: center;
+  font-weight: 700;
+  font-size: 48px;
+  margin-top: 70px;
+  margin-bottom: 40px;
+}
+
+.login_form {
+  padding-left: 450px;
+}
+
+.login_input {
+  width: 535px;
+  border-radius: 7px;
+  border: .006em solid #A9A9A9;
+  font-size: 14px;
+}
+
+input[type="email"] {
+  margin: 20px 0 40px;
+  padding: 30px 20px;
+  color: #A9A9A9;
+  font-weight: 400;
+  font-size: 14px;
+}
+
+input[type="password"] {
+  margin: 20px 0 40px;
+  padding: 30px 20px;
+  color: #A9A9A9;
+  font-weight: 400;
+  font-size: 14px;
+}
+
```

Comment:
The styles for` input[type="email"] `and `input[type="password"]` are the same. You can use a common class for both to avoid duplicating styles and promote reusability.